cfme/tests/test_replication.py

```diff
@@ -342,7 +361,60 @@ def test_replication_global_region_dashboard():
             1.
             2. Dashboard on the Global displays data from the Remote region
     """
-    pass
+    remote_app, global_app = setup_replication
+    remote_provider = provider_app_crud(RHEVMProvider, remote_app)
+    remote_provider.setup()
+    assert remote_provider.name in remote_app.managed_provider_names, "Provider is not available."
+
+    new_vm_name = fauxfactory.gen_alphanumeric(start="test_rep_dashboard", length=25).lower()
+    vm = create_vm(provider=remote_provider, vm_name=new_vm_name)
+    data_items = ('EVM: Recently Discovered Hosts', 'EVM: Recently Discovered VMs',
+                  'Top Storage Consumers')
+
+    remote_dashboard = remote_app.collections.dashboards.instantiate(name="Default Dashboard")
+    remote_view = navigate_to(remote_app.server, "Dashboard")
+    remote_view.browser.refresh()
+    remote_dashboard = remote_view.dashboards("Default Dashboard")
+    remote_app_data = global_app_data = {}
+
+    def get_tabel_data(widget):
+        return [row.name.text for row in widget.contents]
+
+    def data_check(widget):
+        return bool(get_tabel_data(widget))
+
+    for table in data_items:
+        remote_widget = remote_dashboard.widgets(table)
+        wait_for(
+            data_check, func_args=[remote_widget], delay=10, num_sec=900,
+            fail_func=remote_view.browser.refresh,
+            message=f"Waiting for data item of {table} table."
+        )
+        chart_data = get_tabel_data(remote_widget)
+        remote_app_data[table] = chart_data
+
+    global_view = navigate_to(global_app.server, "Dashboard")
+    global_view.browser.refresh()
+    global_dashboard = global_view.dashboards("Default Dashboard")
+    global_app_data = {}
+    for table in data_items:
```

Comment:
**global_app_data** is already declared above: ```remote_app_data = global_app_data = {}```

Reply:
Done